### PR TITLE
Added Mixerwagon as target for shovel mode

### PR DESCRIPTION
**Pull request by schwiti6190**
-Added Mixerwagon, lime and fertilizer spreader and Göweil LT.

### CODE REVIEW

File: scripts/ai/AIDriveStrategyShovelSiloLoader.lua [hunk 1]
```diff
@@ -435,11 +435,20 @@ function AIDriveStrategyShovelSiloLoader:hasTrailerValidSpecializations(trailer)
         --- All normal trailers
         return true
     end
-    if SpecializationUtil.hasSpecialization(Sprayer, trailer.specializations)
-            and trailer.spec_sprayer.isManureSpreader then
+    if SpecializationUtil.hasSpecialization(Sprayer, trailer.specializations) then
+       --     and trailer.spec_sprayer.isManureSpreader then
```
Comment (pvaiko) [inline scripts/ai/AIDriveStrategyShovelSiloLoader.lua:439]: Is this condition intentionally commented out?

File: scripts/ai/AIDriveStrategyShovelSiloLoader.lua [hunk 2]
```diff
@@ -678,7 +694,12 @@ function AIDriveStrategyShovelSiloLoader:approachTrailerForUnloading()
     local firstWpIx = course:getNearestWaypoints(self.vehicle:getAIDirectionNode())
     self:startCourse(course, firstWpIx)
     self:setNewState(self.states.DRIVING_TO_UNLOAD)
-    self.shovelController:calculateMinimalUnloadingHeight(self.targetTrailer.exactFillRootNode)
+    if self.targetTrailer.trailer["spec_pdlc_goeweilPack.balerStationary"] then
+        --- Minimal height calculation is not working for Goeweil balers
+        self.shovelController:setMinimalUnloadingHeight(2.5)
```
Comment (pvaiko) [inline scripts/ai/AIDriveStrategyShovelSiloLoader.lua:699]: Wondering if it is worth to put this into a configuration. Maybe not if it is a single vehicle...

**Review by pvaiko — APPROVED** [review]

File: scripts/pathfinder/PathfinderUtil.lua [hunk 3]
```diff
@@ -240,12 +240,32 @@ end
 ---@class PathfinderUtil.CollisionDetector
 PathfinderUtil.CollisionDetector = CpObject()
 
+--- Nodes of a trigger for example, that will be ignored as collision.
+PathfinderUtil.CollisionDetector.NODES_TO_IGNORE = {}
```
Comment (pvaiko) [inline scripts/pathfinder/PathfinderUtil.lua:244]: Any reason this is uppercase?

File: scripts/ai/AIDriveStrategyShovelSiloLoader.lua (hunk 2 above)
Comment (Tensuko) [reply scripts/ai/AIDriveStrategyShovelSiloLoader.lua:699]: It is just the Göweil stationary baler.
So not worth (yet) to make a configuration for this.
Maybe if we get more tools to unload into that needs to be adjusted.

File: scripts/ai/AIDriveStrategyShovelSiloLoader.lua (hunk 1 above)
Comment (schwiti6190) [reply scripts/ai/AIDriveStrategyShovelSiloLoader.lua:439]: leftover comment will remove it :)

File: scripts/pathfinder/PathfinderUtil.lua (hunk 3 above)
Comment (schwiti6190) [reply scripts/pathfinder/PathfinderUtil.lua:244]: I like to have global/static variables to be uppercase but im not really consistent with it.

**Pull request merged**